dask/tests/test_base.py

```diff
@@ -157,6 +174,15 @@ class NoPickle:
             tokenize(x)
 
 
+@pytest.mark.skipif("not np")
+def test_empty_numpy_array():
+    arr = np.array([])
+    assert arr.strides
+    assert tokenize(arr) == tokenize(arr)
```

Comment:
This looks like something that's useful to push into the tokenize wrapper

Reply:
good call... will update the PR shortly